Library/Homebrew/os/mac/sdk.rb

```diff
@@ -77,7 +77,7 @@ def sdk_paths
 
             Dir[File.join(sdk_prefix, "MacOSX*.sdk")].each do |sdk_path|
               version = sdk_path[/MacOSX(\d+\.\d+)u?\.sdk$/, 1]
-              paths[version] = sdk_path unless version.nil?
+              paths[OS::Mac::Version.new(version)] = sdk_path unless version.nil?
```

Comment:
Can `OS::Mac::Version.new(version)` never be `nil`?

Reply:
No, `OS::Mac::Version.new` always returns a `OS::Mac::Version` instance. Also, this isn't called when `version.nil?` so it will also never be an empty version.

Reply:
Gotcha 👍🏻